package/skelcd-control-SLES.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Aug 11 13:18:10 UTC 2017 - knut.anderssen@suse.com
+
+- Fixed SLES 15 self_update_url (bsc#1051637)
```

Comment:
sorry, but wrong repo. You should update leanos as this skelcd should be used only for autoinstallation.xml